### ISSUE TITLE
Not able to compile the project

### ISSUE THREAD

**Issue by prmondal** [sdkbox/sdkbox-inmobi-sample#16]
I have cloned the repo from master branch. I am trying to invoke "cocos run -p android" which is failing.

../../cocos2d-x-3.6-lib/cocos/./prebuilt-mk/../../prebuilt/android/armeabi/libcocos2d.a:1:1: syntax error, unexpected '!', expecting $end
../../cocos2d-x-3.6-lib/cocos/./prebuilt-mk/../../prebuilt/android/armeabi/libcocos2d.a: not an object or archive

Which NDK version is supported?

**Comment by hugohuang1111** [comment]
do you run 'git submodule update --init' before 'cocos run -p andorid'

**Comment by prmondal** [comment]
Yes. It might be because of latest Cocos console. How can I use cocos console from cocos2d 3.6 for this project?

**Comment by hugohuang1111** [comment]
if you have installed cocos 3.6 framework
run `/Applications/Cocos/frameworks/cocos2d-x-3.6/tools/cocos2d-console/bin/cocos compile -p android`

if you use cocos source code,
run `$cocos_repo_path/tools/cocos2d-console/bin/cocos compile -p android`

**Comment by prmondal** [comment]
I am using cocos version 3.9 and NDK 10d


On Wed, Mar 2, 2016 at 1:07 PM, hugo <notifications@github.com> wrote:

> if you have installed cocos 3.6 framework
> run /Applications/Cocos/frameworks/cocos2d-x-3.6/tools/cocos2d-console/bin/cocos
> compile -p android
>
> if you use cocos source code,
> run $cocos_repo_path/tools/cocos2d-console/bin/cocos compile -p android
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sdkbox/sdkbox-inmobi-sample/issues/16#issuecomment-191112768>
> .
>



-- 
Thanks & Regards,
Prasenjit

**Comment by hugohuang1111** [comment]
please use cocos2d console 3.10 and NDK 10d
